src/components/Person/Person.jsx

```diff
@@ -1 +1,29 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  let partnerMessage = '';
+
+  if (person.isMarried === false) {
```

Comment:
It's better to use switch case statement instead of multiple if else blocks

Reply:
done